src/client/cli/cmd/transfer.cpp

```diff
@@ -119,7 +121,7 @@ QString cmd::Transfer::description() const
 {
     // TODO: Don't mention directories until we support that
     // return QStringLiteral("Copy files and directories between the host and instances.");
-    return QStringLiteral("Copy files between the host and instances.");
+    return QStringLiteral("Transfer files between the host and instances.");
 }
 
 mp::ParseCode cmd::Transfer::parse_args(mp::ArgParser* parser)
```

Comment:
Help text below still needs updating (for the arguments, github does not let me comment).

Reply:
See https://github.com/CanonicalLtd/multipass/pull/789/files/c08d5363ba506133847343450f7a0ea6cd888125..a7e4d27561e5eec365ccf57d63a4fd6ce4c39173#r286850491

Reply:
I updated the help text to include '-', but that is all.